src/App.css

```diff
@@ -0,0 +1,48 @@
+.App {
+  text-align: left;
+  font-family: Roboto;
+  padding-left: 50px;
+  background-color: #282c34;
+  color: white;
+}
+
+.App-logo {
+  height: 40vmin;
+  pointer-events: none;
+}
+
+@media (prefers-reduced-motion: no-preference) {
+  .App-logo {
+    animation: App-logo-spin infinite 20s linear;
+  }
+}
+
+.App-header {
+  background-color: #282c34;
+  min-height: 100vh;
+  display: flex;
+  flex-direction: column;
+  align-items: left;
+  justify-content: cle;
+  font-size: calc(10px + 2vmin);
+  color: white;
+}
+
+.App-link {
+  color: #61dafb;
+}
+
+@keyframes App-logo-spin {
```

Comment:
I believe this is the default React info, I would remove this before starting your project to make sure it doesn't get lost - as it is redundant right now.